### PR TITLE
Department template updates (2)

### PR DESCRIPTION
**Pull request by timbeardshear**

### CODE REVIEW

File: app/views/shared/_feature_slider.html.slim [hunk 1]
```diff
@@ -20,13 +20,11 @@ ruby:
...
                       p = link_to feature.call_to_action_text, feature.call_to_action_link, class: "btn btn-action #{hosting_player(feature.call_to_action_link)}"
             - when "full"
-              .slide
-                .container
-                  - if feature.call_to_action_link && feature.type != "content_only"
-                    a href="#{feature.call_to_action_link}" alt="#{feature.title}"
-                      .segment-graphic.graphic-full style="background-image:url(#{feature.image.url})"
-                  - else
-                    .segment-graphic.graphic-full style="background-image:url(#{feature.image.url})"
+              - if feature.call_to_action_link && feature.type != "content_only"
```
Comment (halloffame) [inline app/views/shared/_feature_slider.html.slim]: @timbeardshear This will still be nested under `.container`, is that what you want?

Comment (timbeardshear) [inline app/views/shared/_feature_slider.html.slim]: @halloffame Yeah, I want that section nested under `.slide` and `.container` but don't want it in there twice. Does that make sense?

Comment (halloffame) [inline app/views/shared/_feature_slider.html.slim]: Yep, that makes sense.

**Comment by halloffame** [comment]
:+1: looks good

**Pull request merged**